hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/TestSecureOzoneCluster.java

```diff
@@ -705,18 +705,9 @@ public void testGetSetRevokeS3Secret() throws Exception {
           OmTransportFactory.create(conf, ugi, null),
           RandomStringUtils.randomAscii(5));
 
-      //Creates a secret since it does not exist
+      // Creates a secret since it does not exist
       S3SecretValue attempt1 = omClient.getS3Secret(username);
 
-      //Fetches the secret from db since it was created in previous step
-      S3SecretValue attempt2 = omClient.getS3Secret(username);
-
-      //secret fetched on both attempts must be same
-      assertEquals(attempt1.getAwsSecret(), attempt2.getAwsSecret());
-
-      //access key fetched on both attempts must be same
-      assertEquals(attempt1.getAwsAccessKey(), attempt2.getAwsAccessKey());
-
```

Comment:
Rather than simply removing the second `getS3Secret()`, we should confirm it now throws expected exception:

```
      // A second getS3Secret on the same username should throw exception
      LambdaTestUtils.intercept(OMException.class,
          "expected error message",
          () -> omClient.getS3Secret(username));
```